client/src/index.css

```diff
@@ -5,6 +5,7 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  white-space: pre-wrap;
```

Comment:
Out of curiosity, what does this line do?

Reply:
It transforms \n into newline.